src/bitmap/backend/atomic_bitmap_mmap.rs

```diff
@@ -0,0 +1,608 @@
+// SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
+
+use crate::bitmap::{Bitmap, BitmapSlice, WithBitmapSlice};
+use crate::mmap::{BitmapMmap, BitmapReplace, NewBitmap};
+use crate::{Address, GuestAddress, GuestUsize};
+use std::ops::Index;
+use std::os::fd::{AsRawFd, BorrowedFd};
+use std::sync::atomic::{AtomicU8, Ordering};
+use std::sync::{Arc, RwLock};
+use std::{io, ptr};
+
+// Size in bytes of the `VHOST_LOG_PAGE`
+const LOG_PAGE_SIZE: usize = 0x1000;
+// Number of bits in an AtomicU8
+const LOG_BLOCK_SIZE: usize = 8;
```

Comment:
I like "word", I'll change that

> The obvious question then becomes why not to use usize instead of u8, but that’s also a question of sharing code with AtomicBitmap, which I ask elsewhere.

I just used u8 because is what qemu used, I don't see why not use usize.  I mentioned in the PR that the plan is to reuse AtomicBitmap